desktop-app/app/components/AddressInput.js

```diff
@@ -312,4 +297,60 @@ class AddressBar extends React.Component<Props> {
 
 const MAX_SUGGESTIONS = 8;
 
-export default AddressBar;
+const styles = theme => {
+  const {mode} = theme.palette;
+  return {
+    ...commonStyles(theme),
+    addressBarContainer: {
+      display: 'flex',
+      position: 'relative',
+      alignItems: 'center',
+      height: '20px',
+      width: '100%',
+      padding: '14px 10px',
+      borderRadius: '20px',
+      backgroundColor: mode({light: theme.palette.grey['300'], dark: 'unset'}),
```

Comment:
Is it possible to standardize these greys with a name? 
So that it can be used consistently?

Reply:
The grey colors come by default in the mui theme: https://material-ui.com/customization/default-theme. 

I think we could standardize a couple if we agree to only use those but if we will use the whole range of grey colors in mui i would prefer to keep the number. It is hard to invent colors' names and projects usually end with: grey, lightGrey, lighterThanLightGrey, .... 😂

Reply:
Yeah right. 😄 

Let's go with this and revisit it later when we have some clarity.